soc/riscv/riscv-privileged/Kconfig

```diff
@@ -28,6 +28,12 @@ config RISCV_HAS_CLIC
 	help
 	  Does the SOC provide support for a Core-Local Interrupt Controller (CLIC).
 
+config RISCV_HAS_CUSTOM_IC
```

Comment:
why not following what is done for ARM and calling this `RISCV_CUSTOM_INTERRUPT_CONTROLLER` ?

Reply:
Hi @carlocaione , 

I started from that naming, but then thought that it doesn't follow the convention used for RISC-V Inerrupt Controllers like RISCV_HAS_PLIC, RISCV_HAS_CLIC, so I decided to follow RISC-V style and not ARM style.

Should we change it to RISCV_CUSTOM_INTERRUPT_CONTROLLER?